File: lectures-master/lectures-master/unit-07-refresher/languages/high-level/high-level.cpp
#include <iostream>
#include <string>
using namespace std;

int countTwos(int num) {
  int count = 0;
  string str = to_string(count);
  for (int i = 2; i <= num; i++) {
    string subStr = to_string(i);
    string::iterator it;
    for (it = subStr.begin(); it < subStr.end(); it++) {
      if (*it == '2') {
        count++;
      }
    }
  }
  return count;
}

int main(int argc, char ** argv) {
  cout << countTwos(100) << endl;
}

// 435 characters typed (not including comment)

File: lectures-master/lectures-master/unit-07-refresher/languages/high-level/high-level.js
function countTwos(num) {
  var count = 0;
  var str = String(num);
  for (var i = 2; i <= num; i++) {
    var subStr = String(i);
    for (var j = 0; j < str.length; j++) {
      if (subStr[j] === '2')
        count++;
    }
  }
  return count;
}

console.log(countTwos(100));

// 278 characters typed (not including comment)
